packages/sdk/src/oracle/store/write.ts

```diff
@@ -81,7 +81,10 @@ export class OptimisticOracle {
   }
   event(event: state.OptimisticOracleEvent): void {
     if (!this.state.events) this.state.events = [];
-    insertOrderedAscending(this.state.events, event, eventKey);
+    // avoid duplicates
+    if (isUnique(this.state.events, event, eventKey)) {
+      insertOrderedAscending(this.state.events, event, eventKey);
+    }
```

Comment:
since there are multiple things writing to these events now, wanted to make sure we dont push in duplicates (though it probably wouldnt matter)